packages/medusa/src/models/line-item.ts

```diff
@@ -72,8 +92,8 @@ export class LineItem extends BaseEntity {
   @Column({ nullable: true })
   description: string
 
-  @Column({ nullable: true })
-  thumbnail: string
+  @Column({ type: "varchar", nullable: true })
```

Comment:
nit(non-blocking) comment info: Just as a general remark, the varchar type should only be used when we want the db system to check for the size of the element, otherwise we should always use text as the type.
I know that varchar without specifying the length restriction is equivalent to TEXT but I think it is more understandable since you often expect to see a limit using varchar ;)

Reply:
@adrien2p Should we revert to not use `varchar` then? Would also keep it consistent with how all other models are defined.

Reply:
Yeah sure 👌